docs/nightly_build.md

```diff
@@ -0,0 +1,54 @@
+# Nightly builds
+
+<!--- TOC -->
+
+* [Configuration](#configuration)
+* [How to register to get nightly build](#how-to-register-to-get-nightly-build)
+* [Build nightly manually](#build-nightly-manually)
+
+<!--- END -->
+
+## Configuration
+
+The nightly build will contain what's on develop, in release mode, for Gplay variant. It is signed using a dedicated signature, and has a dedicated appId (`im.vector.app.nightly`), so it can be installed along with the production version of Element Android. The only other difference compared to Element Android is a different app icon background. We do not want to change the app name since it will also affect some strings in the app, and we do want to do that.
+
+Nightly builds are built and released to Firebase every days, and automatically.
+
+This is recommended to exclusively use this app, with your main account, instead of Element Android, and fallback to Element Android just in case of regression, to discover as soon as possible any regression, and report it to the team. To avoid double notification, you may want to disable the notification from the Element Android production version. Just open Element Android, navigate to `Settings/Notifications` and uncheck `Enable notifications for this session`.
+
+*Note:* Due to a limitation of Firebase, the nightly build is the universal build, which means that the size of the APK is a bit bigger, but this should not have any other side effect.
+
+## How to register to get nightly build
```

Comment:
it's probably worth calling out the tester limit of 500 https://firebase.google.com/docs/app-distribution/add-remove-testers?platform=android#add-remove-testers-group , we'll need to request more slots (if needed) https://console.firebase.google.com/project/_/appdistribution/limits